src/components/Header.js

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
+import { images } from "./images/image";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+  //change parent's user state using props
+  const handleClick = () => {
+    props.onClick();
```

Comment:
props에서 꺼내서 onClick에 넣어준거기때문에 이제 `props.onClick()`이라고 쓸 필요가 없겠죠?

```suggestion
    onClick();
```